go/lambda-api-ci/src/handler.ts

```diff
@@ -0,0 +1,44 @@
+import { S3 } from "aws-sdk"
+
+const bucketName = process.env.BUCKET!
```

Comment:
Wouldn't it make more sense to use Go here or is this out of scope?

Reply:
Might complicate things a bit (especially - you'd have to build that for the correct platform, which is tricky)